### PR TITLE
Install packit in container images from copr

### PR DESCRIPTION
**Pull request by nforro**
Related to packit/packit-service#1601.

Merge after packit/deployment#367.

---

RELEASE NOTES BEGIN
Tooling in container images is now installed from Packit Copr repositories rather than from source using pip.
RELEASE NOTES END

### CODE REVIEW

**Comment by nforro** [comment]
Just a note, sandcastle is still [being installed](https://github.com/packit/packit-service/blob/52fb387b7514d321a9b73bb22355e806a5091ad9/files/install-deps-worker.yaml#L56) from git using pip. Should we do something about it?

File: files/tasks/install-packit-deps.yaml [hunk 1]
```diff
@@ -1,49 +0,0 @@
-# Copyright Contributors to the Packit project.
-# SPDX-License-Identifier: MIT
-
-# Install dependencies for packit
-- name: Download script setupcfg2rpm.py
-  ansible.builtin.get_url:
-    url: https://raw.githubusercontent.com/packit/deployment/main/scripts/setupcfg2rpm.py
```
Comment (jpopelka) [inline files/tasks/install-packit-deps.yaml:7]: Is the `setupcfg2rpm.py` still used anywhere? If not, can it be deleted as well?

**Review by jpopelka — APPROVED** [review]

**Comment by jpopelka** [comment]
> Just a note, sandcastle is still being installed from git using pip. Should we do something about it?

We'd need to package it first and it might now be that easy because there's [one pinned dependency](https://github.com/packit/sandcastle/blob/main/setup.cfg#L31).
And given that we're moving away from building via sandcastle to building in copr, I think it's not worth the effort.

File: files/tasks/install-packit-deps.yaml (hunk 1 above)
Comment (nforro) [reply files/tasks/install-packit-deps.yaml:7]: I don't think it is still used, but there is a reference to it in [tokman containerfile](https://github.com/packit/tokman/blob/8db9af20f24f7e7eb855f25e8431f723c061f955/Containerfile#L6-L7), not sure why.

Comment (nforro) [reply files/tasks/install-packit-deps.yaml:7]: I'm blind, it [is being used](https://github.com/packit/tokman/blob/8db9af20f24f7e7eb855f25e8431f723c061f955/Containerfile#L18) there. Should I keep it or try to remove it?

**Review by TomasTomecek — APPROVED** [review]
nice and clean!! 👋👋

File: files/tasks/install-packit-deps.yaml (hunk 1 above)
Comment (jpopelka) [reply files/tasks/install-packit-deps.yaml:7]: Definitely keep.

**Pull request merged**